content/features/proxy2/run.md

```diff
@@ -0,0 +1,148 @@
++++
+title = 'Run the built-in proxy'
+weight = 40
++++
+
+This page is about running Memcached as a proxy for other Memcached servers.
+
+For a general overview of the proxy, see [Proxy]({{<legacy_proxy_base_path>}}). For a hands-on tutorial that guides you through compiling and running the proxy with a simple configuration, see [Built-in Proxy Quickstart]({{<proxy_base_path>}}quickstart).
+
+## Before you begin {#prereqs}
+
+Before you can run the proxy, you need the following:
+
+* Memcached, with proxy features enabled, installed on your system.
+* The proxy route library file downloaded to some path on your file system.
+* A proxy configuration file that you created.
+
+For more information, see [Install the built-in proxy]({{<proxy_base_path>}}install) and
+[Configure the built-in proxy]({{<proxy_base_path>}}configure).
+
+## Run the proxy
+
+{{< callout type="info" >}}This section assumes that you installed your proxy-enabled Memcached to have the default command name `memcached`. If you configured the build process to name the command something else—for example, `memcached-proxy`—then use that name here instead.{{</callout>}}
+
+To run the proxy, launch the proxy-enabled Memcached binary with the `proxy_config` and `proxy_arg` command-line options:
+
+```
+memcached -o proxy_config={{<var>}}ROUTE_LIBRARY_PATH{{</var>}},proxy_arg={{<var>}}CONFIGURATION_PATH{{</var>}} {{<var>}}FURTHER_ARGUMENTS{{</var>}}
+```
+
+Replace the following:
+
+* <var>ROUTE_LIBRARY_PATH</var>: the file system path to the proxy route library file.
+* <var>CONFIGURATION_PATH</var>: the file system path to your own proxy configuration file.
+* <var>FURTHER_ARGUMENTS</var>: additional Memcached command-line flags. For a complete list, run `memcached --help`.
+
+To run the proxy in the background, append a `&` to the end of the command, in the usual Unix fashion.
+
+### An example command
+
+The following example command launches a Memcached proxy with the following aspects:
+
+* The proxy loads a route library file named `routelib.lua` stored in your home directory.
```

Comment:
Can this hot link to the how to get routelib section somewhere?

Reply:
The "Before you begin" section on this page already states that you need to have downloaded it before any of this will work. That said, there's no harm in clarity, so I've added a little reminder text here; what do you think?

Reply:
Think that's good enough for now. There'll be tons of people who just go to this page directly and get confused at what 'routelib' is, and it's not a link anywhere. So it's good to tell them to go somewhere else.